### ISSUE TITLE
Environment variable ignored when set to own value

### ISSUE THREAD

**Issue by MitMaro** [alexindigo/configly#1]
# Problem

When an environment variable is given it's name as a value, configly ignores the environment variable. I would expect the environment variable to always be used, no matter the value.

### `config/custom-environment-variables.js`

```javascript
'use strict';
module.exports = {
	value: 'VALUE'
};
```

### `config/default.js`

```javascript
'use strict';
module.exports = {
	value: 'FOO'
};
```

### `index.js`

```javascript
'use strict';
console.log(require('configly')().value);
```

[Test case repository](https://github.com/MitMaro/configly-env-error)

### Run

	$ node index.js
	FOO
	$ VALUE=bar node index.js
	bar
	$ # expected output VALUE
	$ VALUE=VALUE node index.js
	FOO

**Comment by alexindigo** [comment]
Hey, thank you for your interest
and for the cool bug report.

I'll definitely look into this :)

**Comment by alexindigo** [comment]
Published `v4.0.2` – https://www.npmjs.com/package/configly
Thank you for the report.
